lib/ts/recipe/session/index.ts

```diff
@@ -24,16 +24,18 @@ import {
     SessionClaim,
     ClaimValidationError,
     RecipeInterface,
+    AntiCsrfType,
 } from "./types";
 import Recipe from "./recipe";
 import { JSONObject } from "../../types";
 import { getRequiredClaimValidators } from "./utils";
 import { createNewSessionInRequest, getSessionFromRequest, refreshSessionInRequest } from "./sessionRequestFunctions";
+import { BaseRequest } from "../../framework";
```

Comment:
If BaseRequest is a type, use `import type`

Reply:
It's not a type